audit-service/src/test/java/uk/gov/gchq/palisade/service/audit/service/StroomAuditServiceTest.java

```diff
@@ -147,7 +146,7 @@ public void auditRegisterRequestUserException() {
                 .withResourceId(resource.getId())
                 .withContext(context)
                 .withException(exception)
-                .withServiceClass(UserService.class);
+                .withServiceName("USER_SERVICE");
```

Comment:
Add the same enum to audit-service too?

Reply:
I agree with @dev930018 comments to add the serviceName enum into the audit service module

Reply:
done